install/css/styles.css

```
@@ -413,15 +437,22 @@ a.bx-btn-disabled,
     font-family: ui-monospace, SFMono-Regular, Menlo, Consolas, monospace !important;
     font-size: 12px;
     color: #374151;
+    overflow-wrap: anywhere;
+    word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)